apps/progs.pl

```diff
@@ -121,6 +133,12 @@
         blake2b512 => "blake2",
         blake2s256 => "blake2",
     );
+    my %md_deprecated = (
+        md2 =>      [ "3_0", "md2" ],
+        md4  =>     [ "3_0", "md4" ],
+        mdc2 =>     [ "3_0", "mdc2" ],
+        rmd160 =>   [ "3_0", "rmd160" ],
+    );
```

Comment:
Er, did you actually mean to have that in this PR?

Reply:
Yes, I intend to push this as a separate commit.
I am willing to make it a separate PR if required, but it's easier here.

Reply:
[Reopening this conversation after seeing comments from Matt on the subject.]

It turns out that these deprecations are premature.